### ISSUE TITLE
Close tab on mouse button release

### ISSUE THREAD

**Issue by kittydoor** [mbnuqw/sidebery#299]
## Description

I would prefer to be able cancel when using the middle mouse button to close tabs. It would be more intuitive to me if a mouse click was the selection, and release was the confirmation (if still located within the bounds of the tab to be closed). Currently, it feels like the tab just disappears while I still have the button pressed down.

**Comment by muesli4** [comment]
Yes, there is no way to opt-out.  This is the way it's implemented in most user interfaces.  If you don't want to change it please add a configurable setting to change it.

**Comment by pintassilgo** [comment]
Agreed. This deviates from the default behavior of all browsers. The default is:

- Switch to tab on mousedown;
 
- Every other action is triggered on click (button release): mute/unmute, close tab button, middleclick to close.

This expected behavior is not honored by Sidebery (except for mute/unmute), so I think there should be an option to revert it.

There's an option `Activate tab on mouse button release`, so it's just about adding the similar `Close tab on mouse button release`.

**Comment by kittydoor** [comment]
I've taken a quick look around, and as per @pintassilgo's comment above, it really doesn't seem that hard to implement! If I work on this, will a pull request be accepted?

**Comment by Qwerty-Space** [comment]
I would also like this feature!  Closing on mouse down is rather annoying
